Sources/BottomSheet/BottomSheetView.swift

```diff
@@ -87,47 +86,45 @@ fileprivate struct BottomSheetView<hContent: View, mContent: View, bottomSheetPo
                 
                 self.mainContent
                     .transition(.move(edge: .bottom))
-                    .animation(Animation.spring(response: 0.5, dampingFraction: 0.75, blendDuration: 1))
```

Comment:
I tested it and it seems to work fine; are you sure that it wount make any problems to remove it?